### PR TITLE
Fixes #247: Add link to show more info about traffic lights (high, lo…

### LINKED ISSUE #247: Add link to show more info about traffic lights (high, low, average)
http://fr.openfoodfacts.org/reperes-nutritionnels

### PR DESCRIPTION
**Pull request by balaji-ramavathu**
…w, average)

## Description
Added Textview to show the link to more info

## Related issues and discussion
fixes #{247}
 
 ## Screen-shots, if any
 
nutri_score_info

### CODE REVIEW

**Comment by balaji-ramavathu** [comment]
@teolemon please review

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -324,6 +324,7 @@
     <string name="txtFat">Fat</string>
     <string name="txtSugars">Sugars</string>
     <string name="hintBarcode">Barcode</string>
+    <string name="txtClickToKnowMore"> Click to know more </string>
```
Comment (teolemon) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="txtClickToKnowMore">How is it computed ?</string>
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -330,6 +330,7 @@
     <string name="txtFat">Fat</string>
     <string name="txtSugars">Sugars</string>
     <string name="hintBarcode">Barcode</string>
+    <string name="txtNutriScoreInfo"> How is it computed ?</string>
```
Comment (teolemon) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="txtNutriScoreInfo">How is it computed ?</string>
```

**Comment by balaji-ramavathu** [comment]
@teolemon any update on this PR?

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/nutrition/NutritionProductFragment.java [hunk 3]
```diff
@@ -284,6 +291,22 @@ public void refreshView(State state) {
...
 
+        textNutriScoreInfo.setClickable(true);
+        textNutriScoreInfo.setMovementMethod(LinkMovementMethod.getInstance());
+        SpannableStringBuilder spannableStringBuilder=new SpannableStringBuilder();
+        ClickableSpan clickableSpan=new ClickableSpan() {
+            @Override
+            public void onClick(@NonNull View view) {
+                CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder().build();
+                customTabsIntent.intent.putExtra("android.intent.extra.REFERRER", Uri.parse("android-app://" + getActivity().getPackageName()));
+                CustomTabActivityHelper.openCustomTab(getActivity(), customTabsIntent, Uri.parse("http://world.openfoodfacts.org/nutrient-levels"), new WebViewFallback());
```
Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/nutrition/NutritionProductFragment.java]: Can you extract the URL to a translatable string ?

Comment (balaji-ramavathu) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/nutrition/NutritionProductFragment.java]: yeah okay

**Comment by brandenfung** [comment]
Looks good.

**Pull request merged**